app/src/backend/formulas.ts

```diff
@@ -466,11 +571,18 @@ export function handleIndustrialWasteWaterFormula(
   return [{ gas: "CH4", amount }];
 }
 
+// TODO how correct is this formula ?
```

Comment:
Yes I think Mau's calculations confirmed we have some work to do on it. We will have tickets for the upcoming sprint for it.